src/core/lib/gprpp/time.h

```diff
@@ -200,6 +200,7 @@ class Duration {
     }
     return *this;
   }
+  Duration& operator*=(double multiplier);
```

Comment:
consider exercising this in `gprpp/time_test.cc`

Reply:
Done.